src/AppInstallerCLICore/ContextOrchestrator.cpp

```diff
@@ -10,6 +10,21 @@
 
 namespace AppInstaller::CLI::Execution
 {
+    namespace
+    {
+        // Callback function used by worker threads in the queue.
+        // context must be a pointer to a queue item.
+        void OrchestratorQueueWorkCallback(PTP_CALLBACK_INSTANCE, PVOID context, PTP_WORK)
+        {
+            auto queueItem = static_cast<OrchestratorQueueItem*>(context);

```

Comment:
I prefer `reinterpret_cast` in these cases to make it clearer that this is not type safe.